frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -184,10 +182,7 @@ export default {
         this.closeModal()
       } catch (e) {
         console.error('setPaymentInfo', e)
-        this.ephemeral.formFeedbackMsg = {
-          text: L(`Failed to update income details. Error: ${e.message}`),
-          class: 'has-text-danger'
-        }
+        this.$refs.formMsg.danger(`${L('Something went wrong, please try again.')} ${e.message}`)
```

Comment:
Much cleaner 😄

Reply:
Though here and in a few other places, can you change the way this text is generated to be [more descriptive and using the L-function variables](https://github.com/okTurtles/group-income-simple/pull/779/files#r349846015)?

I.e. here we can do: `${L('Something went wrong, please try again.')} ${e.message}` => `L('Error updating payment info: {message}, { message: e.message })`

This will make it easier for translators to move the error message if necessary.

Reply:
I did `Something went wrong, please try again.` following @mmbotelho designs. Maybe she can review it. However, I think both messages mean exactly the same thing to the user - _nothing in concrete_. :/ 

I don't see the advantage of placing `e.message` inside the translation because it has nothing to do with the copy itself.

> This will make it easier for translators to move the error message if necessary.

Trying to predict the future...  Anyway, I highly doubt that the code error will be placed somewhere else. 100% of the places I saw this around the web, is the last part of an error message, after the "human message". When we need to do that (if needed), we change the code. Until then I would leave it as it is, simple.

Reply:
> Trying to predict the future...

My job is to predict, and also invent, the future. :)

I think this change needs to be made for the following reasons:

1. If it's not made, we will have inconsistencies in how we translate strings. The code should be consistent. Having a consistent way of translating strings is important, because it helps prevent confusion and makes it easier to onboard new developers.
2. The existing way we have of placing variables into strings is more correct than what's being done here, because it gives our translators more flexibility
3. As far as copy goes, the copy suggested above is better than what was there before. "Something went wrong" is a bad error message.